clients/src/main/java/org/apache/kafka/common/config/ConfigResource.java

```diff
@@ -33,7 +33,7 @@
      * Type of resource.
      */
     public enum Type {
-        BROKER_LOGGER((byte) 8), BROKER((byte) 4), TOPIC((byte) 2), UNKNOWN((byte) 0);
+        CLIENT_METRICS((byte) 16), BROKER_LOGGER((byte) 8), BROKER((byte) 4), TOPIC((byte) 2), UNKNOWN((byte) 0);
 
```

Comment:
To make it work you need the changes from the PR: https://github.com/confluentinc/kafka/pull/635